runtime/doc/lsp.txt

```diff
@@ -1981,11 +1981,8 @@ start({cmd}, {cmd_args}, {dispatchers}, {extra_spawn_params})
         Methods:
         • `notify()` |vim.lsp.rpc.notify()|
         • `request()` |vim.lsp.rpc.request()|
-
```

Comment:
start() docs also still mention:

> Communication with the server is currently limited to stdio.

Reply:
Thanks for pointing out. I opened https://github.com/neovim/neovim/pull/20120